packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -768,6 +791,21 @@ export class ActiveWorkflowRunner {
 		}
 	}
 
+	/**
+	 * Stop running active triggers and pollers for a workflow.
+	 */
+	async removeWorkflowTriggersAndPollers(workflowId: string) {
```

Comment:
Maybe we can deduplicate the `remove` function above to use this function as well wdyt?